File: 03) Software Technologies/30) EXAM HELPERS/Java/Entity.java
package project.entity;

import javax.persistence.*;

@Entity
@Table(name = "entities")
public class Entity {
    public Entity() {
    }

    public Entity(String stringProperty, int intProperty) {
	this.stringProperty = stringProperty;
	this.intProperty = intProperty;
    }

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int Id;

    @Column(nullable = false)
    private String stringProperty;

    @Column(nullable = false)
    private int intProperty;

    @Column(nullable = false)
    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    @Column(nullable = false)
    public String getStringProperty() {
        return stringProperty;
    }

    public void setStringProperty(String stringProperty) {
        this.stringProperty = stringProperty;
    }
	
    @Column(nullable = false)
    public int getIntProperty() {
        return intProperty;
    }

    public void setIntProperty(int intProperty) {
        this.intProperty = intProperty;
    }
}

File: 03) Software Technologies/30) EXAM HELPERS/PHP/Entity.php
<?php
namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Entity
 *
 * @ORM\Table(name="entities")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\EntityRepository")
 */
class Entity
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;
	
    /**
     * @var string
     *
     * @ORM\Column(name="stringProperty", type="string", length=255)
     */
    private $stringProperty;	
	
	/**
     * @var int
     *
     * @ORM\Column(name="intProperty", type="integer")
     */
    private $intProperty;
	
    /**
     * @return int
     */
    public function getId(): ?int
    {
        return $this->id;
    }

//    /**
//     * @param int $id
//     */
//    public function setId(?int $id): Entity
//    {
//        $this->id = $id;
//        return $this;
//    }

    /**
     * @return string
     */
    public function getStringProperty(): ?string
    {
        return $this->stringProperty;
    }
	
    /**
     * @param string $stringProperty
     * @return Entity
     */
    public function setTitle(?string $stringProperty): Entity
    {
        $this->stringProperty = $stringProperty;
        return $this;
    }	

    /**
     * @return int
     */
    public function getIntProperty(): ?int
    {
        return $this->intProperty;
    }

    /**
     * @param int $intProperty
     * @return Entity
     */
    public function setYear(?int $intProperty): Entity
    {
        $this->year = $intProperty;
        return $this;
    }
	
}